pkg/overlay/overlay.go

```diff
@@ -93,6 +106,12 @@ func mountHelper(contentDir, source, dest string, _, _ int, graphOptions []strin
 		// Read-write overlay mounts want a lower, upper and a work layer.
 		workDir := filepath.Join(contentDir, "work")
 		upperDir := filepath.Join(contentDir, "upper")
+
+		if opts != nil {
+			workDir = opts.Workdir
```

Comment:
should we set these only if the opts version is != "" ?

Reply:
I agree, we need to be able to add more `Options` members without forcing callers to choose a `Workdir` etc.